targets/science/include/PinNames.h

```diff
@@ -178,8 +185,9 @@ typedef enum {
   LIM_SW_4 = PC_12,
 
   /**** Encoder Pins ****/
-  ENC_DCI_A = PB_6,
-  ENC_DCI_B = PB_7,
+  ENC_DCI_A  = PB_6,
+  ENC_DCI_B  = PB_7,
+  ENC_SCI_IN = PB_5,
```

Comment:
above

Reply:
no this was there from the beginning, I can remove it I dont think it does anything

Reply:
nah if this was always there dont remove it. actually since this was never merged into master, def leave it

Reply:
sounds good

Reply:
should double check against the latest science schematic too. Maybe things moved around etc. Also would be a good time to also make sure the peripherals u have in code matches actual hw lmao. ie. check # of motors match, # and types of encoders, # of servos, # of limit switches etc.